CHANGELOG

```diff
@@ -1,5 +1,9 @@
 devel
 -----
+* fix internal issue #478: implement 'js/actions/_admin/statistics/app.js'
```

Comment:
Not really an accurate description of what happened...
The relevant information for CHANGELOG readers is probably more like 

    removed unused and undocumented REST API endpoints /_admin/statistics/long and /_admin/statistics/short
    
    these APIs were available in ArangoDB's REST API, but not called by ArangoDB itself nor part of the documented API
    as they have been superseded by other REST APIs and were partly dysfunctional, these two endpoints have been removed entirely.